.travis.yml

```diff
@@ -34,4 +31,5 @@ script:
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_snmp_information
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_users
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_traceroute
+- nosetests -v TestNXOSDriver:TestNXOSDriver_bgp_neighbors
```

Comment:
@dbarrosop, I tried to combine them, but then realized that the tests that I put together can only be used when `cls.mock = True`, since the methods force some fake return values.  We could use a mocking framework, https://docs.python.org/3/library/unittest.mock.html, or something like that, but that felt like an invasive change so I didn't want to introduce that.